framework/src/modules/chain/chain.js

```diff
@@ -152,6 +155,19 @@ module.exports = class Chain {
 			this.scope.bus = await createBus();
 			this.scope.logic = await initLogicStructure(this.scope);
 			this.scope.modules = await initModules(this.scope);
+			// TODO: Global variable forbits to require on top
+			const Loader = require('./loader');
+			const { Forge } = require('./forge');
+			const { Delegates } = require('./submodules/delegates');
+			const Transport = require('./transport');
+			this.loader = new Loader(this.scope);
+			this.forge = new Forge(this.scope);
+			this.transport = new Transport(this.scope);
+			// TODO: should not add to scope
+			this.scope.modules.delegates = new Delegates(this.scope);
+			this.scope.modules.loader = this.loader;
+			this.scope.modules.forge = this.forge;
+			this.scope.modules.transport = this.transport;
```

Comment:
If these are modules then what's wrong to load and return from in `init_modules` step?

Reply:
It's still temporally, and I want to remove this `init_modules` step afterwards